frontend-implicaction/src/app/user/components/user-profile/user-profile.component.ts

```diff
@@ -14,9 +15,12 @@ import {Subscription} from 'rxjs';
 export class UserProfileComponent implements OnInit, OnDestroy {
 
   user: User = {};
```

Comment:
```suggestion
  user = {} as User;
```
voir si c'est pas mieux